<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>150</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CreateSWF</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QMenuBar" name="_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="_menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="_menuRecentFolders">
     <property name="title">
      <string>&amp;Recent Folders</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/16x16/open.png</normaloff>:/images/16x16/open.png</iconset>
     </property>
     <addaction name="_actionClearRecentFolders"/>
    </widget>
    <addaction name="_actionOpen"/>
    <addaction name="_actionPreferences"/>
    <addaction name="_menuRecentFolders"/>
    <addaction name="separator"/>
    <addaction name="_actionQuit"/>
   </widget>
   <widget class="QMenu" name="_menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="_actionAbout"/>
    <addaction name="_actionAboutQt"/>
   </widget>
   <addaction name="_menuFile"/>
   <addaction name="_menuEdit"/>
   <addaction name="_menuView"/>
   <addaction name="_menuMap"/>
   <addaction name="_menuHelp"/>
  </widget>
  <widget class="QToolBar" name="_toolbar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="_actionOpen"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QWidget" name="temp">
   <layout class="QVBoxLayout" name="_vbLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="_listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="_gridLayoutList">
       <item row="0" column="0">
        <widget class="QWidget" name="_grpList">
         <layout class="QGridLayout" name="_gridLayoutListInner">
          <property name="alignment">
           <set>Qt::AlignLeft|Qt::AlignTop</set>
          </property>
        <item row="0" column="0">
         <widget class="QListView" name="_dirList">
          <property name="geometry">
           <rect>
            <height>10</height>
           </rect>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
     <widget class="QWidget" name="_btnContainer">
      <layout class="QGridLayout" name="_gridLayoutBtnInner">
       <item row="0" column="0">
        <widget class="QPushButton" name="_buttonAdd">
         <property name="text">
          <string>Add Directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="_buttonRemove">
         <property name="text">
          <string>Remove Directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="_buttonCompile">
         <property name="text">
          <string>Compile</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  </item>
  </layout>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <action name="_actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16x16/open.png</normaloff>:/images/16x16/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="_actionQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16x16/open.png</normaloff>:/images/16x16/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="_actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16x16/open.png</normaloff>:/images/16x16/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About CreateSWF</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="_actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="_actionPreferences">
   <property name="text">
    <string>Pre&amp;ferences...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="_actionClearRecentFolders">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/16x16/open.png</normaloff>:/images/16x16/open.png</iconset>
   </property>
   <property name="text">
    <string>Clear Recent Folders</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
